lake/top/lake_top.py

```diff
@@ -357,7 +357,7 @@ def __init__(self,
         self.wire(cfg_seq_clk, kts.util.clock(self._gclk))
 
         self.add_child(f"config_seq", stg_cfg_seq,
-                       clk=self._cfg_seq_clk,
+                       clk=self._gclk,
```

Comment:
Why did you change this? These actually have a different clock - you can't write to the SRAM at configuration time with this different clock...

Reply:
I changed it because there were issues with having 2 clocks for one of the scripts Nestan was using, but we ended up not using that anyway, so I will revert it.

Reply:
Just pushed, but isn't cfg_seq_clk just connected to gclk anyway? Does this make a difference?